packages/components/src/tabs/README.md

```diff
@@ -1,254 +1,216 @@
-			onSelect={ ( selectedId ) => {
-				setSelectedTabId( selectedId );
-				onSelect( selectedId );
-			} }
-		>
-			<Tabs.TabList>
-				<Tabs.Tab tabId="tab1" title="Tab 1">
-					Tab 1
-				</Tabs.Tab>
-				<Tabs.Tab tabId="tab2" title="Tab 2">
-					Tab 2
-				</Tabs.Tab>
-				<Tabs.Tab tabId="tab3" title="Tab 3">
-					Tab 3
-				</Tabs.Tab>
-			</Tabs.TabList>
-			<Tabs.TabPanel tabId="tab1">
-				<p>Selected tab: Tab 1</p>
-			</Tabs.TabPanel>
-			<Tabs.TabPanel tabId="tab2">
-				<p>Selected tab: Tab 2</p>
-			</Tabs.TabPanel>
-			<Tabs.TabPanel tabId="tab3">
-				<p>Selected tab: Tab 3</p>
-			</Tabs.TabPanel>
-		</Tabs>
-	);
-```
-
-### Components and Sub-components
-
-Tabs is comprised of four individual components:
-- `Tabs`: a wrapper component and context provider. It is responsible for managing the state of the tabs and rendering the `TabList` and `TabPanels`.
-- `TabList`: a wrapper component for the `Tab` components. It is responsible for rendering the list of tabs.
-- `Tab`: renders a single tab. The currently active tab receives default styling that can be overridden with CSS targeting [aria-selected="true"].
-- `TabPanel`: renders the content to display for a single tab once that tab is selected.
-
-#### Tabs
-
-##### Props
-
-###### `children`: `React.ReactNode`
-
-The children elements, which should include one instance of the `Tabs.Tablist` component and as many instances of the `Tabs.TabPanel` components as there are `Tabs.Tab` components.
-
--   Required: Yes
-
-###### `selectOnMove`: `boolean`
-
-Determines if the tab should be selected when it receives focus. If set to `false`, the tab will only be selected upon clicking, not when using arrow keys to shift focus (manual tab activation). See the [official W3C docs](https://www.w3.org/WAI/ARIA/apg/patterns/tabpanel/) for more info.
-
--   Required: No
--   Default: `true`
-
-###### `selectedTabId`: `string | null`
+<!-- This file is generated automatically and cannot be edited directly. Make edits via TypeScript types and TSDocs. -->
 
-The id of the tab whose panel is currently visible.
+<p class="callout callout-info">See the <a href="https://wordpress.github.io/gutenberg/?path=/docs/components-tabs--docs">WordPress Storybook</a> for more detailed, interactive documentation.</p>
 
-If left `undefined`, it will be automatically set to the first enabled tab, and the component assumes it is being used in "uncontrolled" mode.
+Tabs is a collection of React components that combine to render
+an [ARIA-compliant tabs pattern](https://www.w3.org/WAI/ARIA/apg/patterns/tabs/).
 
-Consequently, any value different than `undefined` will set the component in "controlled" mode. When in "controlled" mode, the `null` value will result in no tabs being selected, and the tablist becoming tabbable.
+Tabs organizes content across different screens, data sets, and interactions.
+It has two sections: a list of tabs, and the view to show when tabs are chosen.
```

Comment:
Only one tab can be chosen at a given time, so I think it makes sense to reflect that more accurately. 

```suggestion
It has two sections: a list of tabs, and the view to show when a tab is chosen.
```